components/album/SubAlbum.vue

```diff
@@ -64,6 +64,8 @@ const onResume = () => {
       </div>
 
       <div v-if="active && album.children" class="ml-auto flex gap-2">
+        <AlbumMenu :album="album"></AlbumMenu>
+        <!-- ToDo: this is currently not clickable since we're not preventing event bubbling -->
```

Comment:
@sifferhans When clicking the 3 dots, it closes the subalbum instead of opening the menu. Not sure how to prevent the event from bubbling. Can you have a look?
If it's not quickly possible, I'll just remove it for subalbums.